install/README.md

```diff
@@ -219,6 +220,47 @@ A list of _datalogger_ sampling configurations for a given _station_ and recordi
 | _Start_ | Stream start time|
 | _Stop_ | Stream stop time|
 
+#### _POLARITIES_ ####
+
+Site specific times when the recorded values may have a reversed, or otherwise, polarity.
+This is often difficult to track with the known instrument responses and installation details as it can
+often be caused by in-field wiring or cabling changes from the expected standard.
+
+There is also the possibility that there may be conflicting information or studies, where this is
+the case the _Primary_ field can be used to indicate which one should be used for downstream processing.
+
+To reduce the number valid _Method_ entries these are defined as a set of "known" values, and an "unknown" one.
+Where possibly the code should be updated to add any standard polarity detection methods, there is also
+room to provide a reference citation for any associated studies.
+
+
+| Field | Description | Units |
+| --- | --- | --- |
+| _Station_ | Recording _Station_|
+| _Location_ | Recording location _Site_|
+| _Sublocation_ | Recording location _Sublocation_|
+| _Subsource_ | Recording location _Subsource_|
+| _Primary_ | Whether the entry takes precedence| _"yes"_, _"no"_, or blank.
+| _Reversed_ | Whether the stream in the time window should be considered reversed or not| _"yes"_ or _"no"_
+| _Method_ | How this information was obtained| _"study"_, _"compass"_, or _"unknown"_
```

Comment:
Some elaboration on what this means could be useful

Reply:
i.e., "compass" shows sensor alignment, and if data are reversed then polarity...

Reply:
Also, is this where the references files (https://github.com/GeoNet/delta/pull/1424/files) could be included under a citation column?

Reply:
Method could also go in long-form but high-level documentation, as we have for orientations.